documentation/topics/advanced/pagination.livemd

```diff
@@ -895,9 +899,45 @@ last_page.more?
 false
 ```
 
-### Actions supporting both offset and keyset pagination
+### Default Pagination Behavior When Both Types Are Supported
+
+When an action supports both `offset` and `keyset` pagination (such as default read actions), Ash uses the following logic to determine which pagination type to use:
+
+**1. Explicit pagination parameters take precedence:**
+- If `after` or `before` is provided → keyset pagination
+- If `offset` is provided → offset pagination
+
+**2. Configuration-based default for ambiguous cases:**
+When only `limit` is provided, Ash checks your application configuration:
+
+```elixir
+# Check your config/config.exs
+config :ash, :default_page_type, :keyset  # or :offset
+```
+
+- If `config :ash, :default_page_type, :keyset` → returns `Ash.Page.Keyset`
+- If `config :ash, :default_page_type, :offset` → returns `Ash.Page.Offset`
+- If no configuration is set → defaults to `Ash.Page.Offset`
 
-If an action supports both offset and keyset pagination (e.g. default read actions), offset pagination is used by default when page options only contain `limit`. However, the records will have the keyset in the metadata, so keyset pagination can be performed on next pages.
+> #### ⚠️ Configuration Warning
+>
+> Never set `config :ash, :default_page_type, nil` as this will cause a runtime crash.
```

Comment:
This warning is a bit specific. I'd suggest trimming down these docs a bit WRT the configuration and instead just link to the backwards compatbility config guide. No need to explain both options here. In this case this reads more like a "feature" rather than a backwards compatibility tool.